shared/network/2.x.md

```diff
@@ -508,6 +508,32 @@ You can configure a connectivity check using NetworkManager's builtin [feature][
 custom endpoints, the URL, expected response, and check interval can be set in `config.json` (more information available
 [here][meta-balena-connectivity]). By default, NetworkManager is configured to use {{ $names.cloud.lower }} to perform the check.
 
+## Disable IPv6
+
+In cases where a local network issue is preventing IPv6 traffic from being routed, you can fully disable IPv6 in {{ $names.os.lower }} 2.0 with the following commands.
+
+__Warning:__ Making changes to the networking of a device is extremely dangerous and can lead to a device being unrecoverable, so exercise caution with any of the following.
+
+```
+root@087fd7c:/# nmcli con show
+NAME                UUID                                  TYPE      DEVICE
+Wired connection    087fd7cd-9a6b-423c-8b64-c30cbf6a7126  ethernet  eno1
+...
+
+root@087fd7c:~# nmcli con edit 087fd7cd-9a6b-423c-8b64-c30cbf6a7126
+
+nmcli> set ipv6.method disable
+
+nmcli> save persistent
+Connection 'Wired connection' (087fd7cd-9a6b-423c-8b64-c30cbf6a7126) successfully updated.
+
+nmcli> quit
+
+root@087fd7c:~# nmcli con down 087fd7cd-9a6b-423c-8b64-c30cbf6a7126; nmcli con up 087fd7cd-9a6b-423c-8b64-c30cbf6a7126
```

Comment:
I am pretty sure `nmcli con up ...` will do the implicit down if there are changes, so we can shorten this part.

Reply:
> I am pretty sure `nmcli con up` ... will do the implicit down if there are changes, so we can shorten this part.

@majorz can you confirm this is the case with the last few versions of NM we shipped?

Reply:
@klutchell Yes, it will do an implicit down indeed.

Also we can shorten all `nmcli con` calls to just `nmcli c`.